pkgs/games/en-croissant/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, lib
+, dpkg
+, fetchurl
+, autoPatchelfHook
+, wrapGAppsHook
+, webkitgtk
+, libsoup
+, openssl_1_1
+}:
+
+stdenv.mkDerivation rec {
+  name = "en-croissant";
+  version = "0.7.1";
+
+  src = fetchurl {
+    url = "https://github.com/franciscoBSalgueiro/en-croissant/releases/download/v${version}/en-croissant_${version}_amd64.deb";
+    sha256 = "1lrl0pg0aihx87yqygfailbf9n9ra1l0a7zg40j36c92h8k0zmcb";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+  ];
+
+  buildInputs = [
+    openssl_1_1
```

Comment:
openssl_1_1 is deprecated and in the process of being removed from nixpkgs https://github.com/NixOS/nixpkgs/issues/269713